{
  "id": "MAL-2024-6139",
  "summary": "Malicious code in threadlimite (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:43:36Z",
  "published": "2024-06-25T13:43:36Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:43:36Z",
        "import_time": "2024-06-28T02:51:00.786127193Z",
        "versions": [
          "0.2",
          "0.1"
        ],
        "id": "RLMA-2024-04943",
        "source": "reversing-labs",
        "sha256": "0e9527b6465fecb9a065a380abfa77e77d29ebe4d0af3d18883a4832e8a4d6de"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "threadlimite",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/threadlimite"
      },
      "versions": [
        "0.2",
        "0.1"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/threadlimite/MAL-2024-6139.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}